### ISSUE TITLE
BUG: groupmeans fails to work when data is read from csv

### ISSUE THREAD

**Issue by skathirmani** [gramener/autolysis#8]
For example, `suicide.csv` data throws the following error

```
C:\Anaconda3\lib\site-packages\pandas\core\common.py in _asarray_tuplesafe(values, dtype)
   2009             except ValueError:
   2010                 # we have a list-of-list
-> 2011                 result[:] = [tuple(x) for x in values]
   2012 
   2013     return result

ValueError: cannot copy sequence with size 7 to array axis with dimension 306

```

**Comment by pratapvardhan** [comment]
Can you provide the complete log @skathirmani please?

**Comment by pratapvardhan** [comment]
fixed via #16 and tested in #20
